thinkhazard/templates/report.jinja2

```diff
@@ -166,11 +165,29 @@
 {% endblock %}
 
 {% block scripts %}
-  <script>
-    {% if hazard %}
-      app.hazardType = '{{hazard.hazardtype.mnemonic}}';
-    {% endif %}
-  </script>
+{% if debug %}
+    <script src="{{('%s/openlayers/dist/ol-debug.js' % node_modules)|static_url}}"></script>
+{% else %}
+    <script src="{{('%s/openlayers/dist/ol.js' % node_modules)|static_url}}"></script>
+{% endif %}
+    <script>
+      {% if hazard %}
```

Comment:
I'd suggest using `{%- if hazard -%}` everywhere to avoid blank lines. See http://jinja.pocoo.org/docs/dev/templates/#whitespace-control.